### PR TITLE
feat: new way of versioning beta builds

### PR DESCRIPTION
**Pull request by artsyjian**
[setuptools_scm](https://github.com/artsy/hokusai/pull/330) was introduced around a year ago to help version Beta builds. [Our experience](https://github.com/artsy/hokusai/issues/384) shows that it is not worth the trouble.

This PR removes setuptools_scm, restores hokusai/VERSION file and much of the pre-setuptools_scm configs, and introduces a different way of versioning Beta builds.

### CODE REVIEW

File: hokusai/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+999.999.999
```
Comment (artsyjian) [inline hokusai/VERSION:1]: not 0.0.0 b/c [some artsy projects](https://github.com/artsy/gravity/blob/caf28fd0345e50ae928d66404e3b15fc0b902707/hokusai/config.yml#L5) have min Hokusai version requirements.

File: hokusai/lib/environment.py [hunk 2]
```diff
@@ -26,3 +26,9 @@ def templates_dir_path():
     return sys._MEIPASS + '/hokusai_datas/templates'
   else:
     return os.path.dirname(hokusai.__file__) + '/templates'
+
+def version_file_path():
+  if frozen():
+    return sys._MEIPASS + '/hokusai_datas/VERSION'
+  else:
+    return os.path.dirname(hokusai.__file__) + '/VERSION'
```
Comment (artsyjian) [inline hokusai/lib/environment.py:34]: hokusai/VERSION file path differs depending on the build, PyInstaller build vs. Pip vs. local

**Comment by artsyjian** [comment]
Should resolve https://github.com/artsy/hokusai/issues/384 and https://github.com/artsy/hokusai/issues/357.

File: hokusai/VERSION (hunk 1 above)
Comment (artsyjian) [reply hokusai/VERSION:1]: it's just a placeholder

Comment (mc-jones) [reply hokusai/VERSION:1]: Minor: could be worth adding this context as a comment as it is non-obvious.

**Review by mc-jones — APPROVED** [review]
Nice change as I've been bit by the versioning issue before. 👍

File: hokusai/VERSION (hunk 1 above)
Comment (artsyjian) [reply hokusai/VERSION:1]: True. But a comment would complicate [the code that reads the file](https://github.com/artsy/hokusai/pull/389/files#diff-c289bc883a0cbdc59ef79744e976404261024f587ca9a8b3af08e1396da946a8R6). How about [adding the info to README](https://github.com/artsy/hokusai/pull/389/commits/e59d52944a88e7a82d62968e48e0488b623bfb28)?

Comment (mc-jones) [reply hokusai/VERSION:1]: That works!

**Pull request merged**